ui/js/models/perf/alerts.js

```diff
@@ -75,25 +75,56 @@ treeherder.factory('PhAlerts', [
                 this.updateStatus(status);
             };
         });
-        AlertSummary.prototype.getTextualSummary = function() {
+        AlertSummary.prototype.getTextualSummary = function(copySummary = false) {
             var resultStr = "";
             var improved = _.filter(this.alerts, function(alert) {return !alert.is_regression && alert.visible;});
             var regressed = _.filter(this.alerts, function(alert) {return alert.is_regression && alert.visible;});
+            // add summary header if getting text for clipboard only
+            if (copySummary) {
+                var lastUpdated = new Date(this.last_updated)
+                resultStr += "== Change summary for alert #" + this.id +
+                             " (as of " + lastUpdated.toLocaleFormat("%B %d %Y %H:%M UTC") + ") ==\n"
+            }
             if (regressed.length > 0) {
-                resultStr += "Summary of tests that regressed:\n\n" +
-                             _.map(regressed, function(alert) {
-                                 return "  " + alert.title + " - " + alert.amount_pct + "% worse";
-                             }).join('\n') + "\n";
+                // more details when getting text for the clipboard
+                if (copySummary) {
```

Comment:
I think we can unconditionally use the new syntax for the alerts themselves, as I think the syntax is a bit nicer. We just don't need the header.